docs/Community/ReleaseProcess.md

```diff
@@ -0,0 +1,51 @@
+Open Enclave SDK Release Process
+================================
+
+(TODO - Update this)
+
+Community Approval of Releases
+------------------------------
+
+We want to ensure that our releases go through a process of community feedback.
+That means that before any release is finalized, a feedback solicitation period
+will happen. The release manager will open a GitHub issue announcing a Release
```

Comment:
the term "release manager" is used here without reference or definition.
I have previously asked "Who is the release manager and how are they selected"
I *think* from your PR that the answer in your terminology might be "the lead of the Release SIG" in which case we should consider using that term instead

Reply:
This section is simply copied from previous documentation.

Reply:
We previously voted on the release manager in CGC. The release manager is currently me(after John left)

Reply:
@radhikaj Agreed, I'm imagining that we label it as a Release SIG with you as the SIG lead, and any public release meeting you run would be a Release SIG meeting.  Then we don't need to treat "voting on a release manager" as any separate process if you step down in the future, it'd be the same as for any other SIG.